src/seedu/addressbook/Main.java

```diff
@@ -78,16 +79,33 @@ private void exit() {
 
     /** Reads the user command and executes it, until the user issues the exit command.  */
     private void runCommandLoopUntilExitCommand() {
-        Command command;
+        Command command = null;
         do {
-            String userCommandText = ui.getUserCommand();
-            command = new Parser().parseCommand(userCommandText);
-            CommandResult result = executeCommand(command);
-            recordResult(result);
-            ui.showResultToUser(result);
-
+        	try {
+                String userCommandText = ui.getUserCommand();
+                checkStorageFileExistence();
```

Comment:
Good: added a method to capture the commonly used code.

However, there may be better place for this check, e.g. in the "save()" method. Try to read this method as a outsider and this new check stood out (i.e. breaks SLAP). "Why am I checking storage file here as I have just gotten the user command the statement before?)